packages/synapse-react-client/src/components/widgets/RangeSlider/RangeSlider.tsx

```diff
@@ -14,8 +14,8 @@ export type RangeSliderProps = React.PropsWithChildren<{
   domain: string[]
   initialValues: RangeValues
   step: number
-  doUpdateOnApply?: boolean
-  onChange: (values: RangeValues) => void
+  onChange?: (values: RangeValues) => void
+  onApplyClicked?: (values: RangeValues) => void
```

Comment:
Removed `doUpdateOnApply`, which decided whether `onChange` should be invoked "on change" or "on apply clicked".

Instead, add `onApplyClicked`. Always call each callback when the user does what the callback describes.